media/js/src/wizard.js

```diff
@@ -51,9 +88,17 @@
                 ul.append(pb);
             }
             if (index < (length - 1)) {
-                var nb = $('<li></li>').append($('<a>Next &gt;</a>')
-                    .attr({href:'#'}));
-                nb.click(function() {
+                var a = $('<a>Next &gt;</a>')
+                    .attr({href:'#', class:'btn next-button',
+                           id:'next-button-' + index});
+                if (index != 1) {
```

Comment:
Also, this should be `!==` instead of `!=`. In general I try and use the equality operators that don't do automatic type conversion, because the type conversion can give unexpected behavior and is an extra thing to think about.